src/hooks/sdk/useSelectVideoQuality.tsx

```diff
@@ -20,13 +20,16 @@ export function useSelectVideoQuality() {
 
       switch (quality) {
         case '360p':
-          audioVideo.chooseVideoInputQuality(640, 360, 15, 600);
+          audioVideo.chooseVideoInputQuality(640, 360, 15);
+          audioVideo.setVideoMaxBandwidthKbps(600);
           break;
         case '540p':
-          audioVideo.chooseVideoInputQuality(960, 540, 15, 1400);
+          audioVideo.chooseVideoInputQuality(960, 540, 15);
+          audioVideo.setVideoMaxBandwidthKbps(1000);
```

Comment:
You do not need to call `audioVideo.setVideoMaxBandwidthKbps(1400);` as it is the default value.

Reply:
I think it's needed here.
If user chooses 360p first, the `VideoMaxBandwidthKbps` is set to 600.
if user chooses 540p then, the `VideoMaxBandwidthKbps` is not updated and still 600.

Reply:
Oops. Need to fix this in JS SDK demo then :D